app/lib/shared/handlers.dart

```diff
@@ -112,6 +112,10 @@ shelf.Response htmlResponse(String content,
   return shelf.Response(status, body: content, headers: headers);
 }
 
+// TODO: create a customized handler for invalid requests
+shelf.Response invalidRequestHandler(shelf.Request request) =>
+    notFoundHandler(request);
```

Comment:
This should ideally be a 400, because you're sending us something that's not a valid request.